data/src/main/java/com/moez/QKSMS/manager/WidgetManagerImpl.kt

```diff
@@ -33,7 +33,7 @@ class WidgetManagerImpl @Inject constructor(private val context: Context) : Widg
 
     override fun updateTheme() {
         val ids = AppWidgetManager.getInstance(context)
-                .getAppWidgetIds(ComponentName("dev.octoshrimpy.quik", "dev.octoshrimpy.quik.feature.widget.WidgetProvider"))
+                .getAppWidgetIds(ComponentName("com.unplugged.messages", "dev.octoshrimpy.quik.feature.widget.WidgetProvider"))
```

Comment:
Why don't we change "dev.octoshrimpy.quik.feature.widget.WidgetProvider"?

Reply:
This is one of the things that break if I change the namespace and it's not user facing so no need to change everything